docs/source/index.rst

```diff
@@ -3,18 +3,58 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to HookMan's documentation!
-===================================
+=================================
+HookMan documentation!
+=================================
+
+This documentation covers HookMan usage & API documentation. 
```

Comment:
This documentation covers HookMan usage & API.

Reply:
Fixed ✔️